contracts/actions/ClosePosition.sol

```diff
@@ -20,9 +20,9 @@ interface IClosePosition {
 
 contract ClosePosition is IClosePosition {
     ///@notice emitted when a UniswapNFT position is closed
-    ///@param from address of PositionManager
+    ///@param positionManager address of PositionManager
     ///@param tokenId Id of the closed token
-    event CloseUniPosition(address indexed from, uint256 tokenId);
+    event CloseUniPosition(address indexed positionManager, uint256 tokenId);
```

Comment:
Still here different, if we had the verbal form in the past, let's have it consistently on all the events

Reply:
or let's choose whichever convention that makes sense for all the events